src/InteractiveSlide/InstantBook.php

```diff
@@ -453,7 +486,7 @@ private function checkPermission(InteractiveSlide $interactive, string $resource
             $allowedResources = $this->getAllowedResources($interactive);
 
             if (!in_array($resource, $allowedResources)) {
-                throw new \Exception('Not allowed');
+                throw new InteractiveSlideException('Not allowed', 403);
```

Comment:
```suggestion
                throw new InteractiveSlidePermissionException('Not allowed');
```
Then create `InteractiveSlidePermissionException` and map it to 403 Forbidden